.gitignore

```diff
@@ -1,3 +1,4 @@
+push_script.py
```

Comment:
you can create in your folder your gitignore https://stackoverflow.com/questions/3719574/is-it-possible-to-ignore-gitignore-rules-in-subdirectory

please remove your line from common gitgnore